index.bs

```diff
@@ -2187,35 +3597,134 @@ dictionary MLLstmOptions {
   sequence<MLActivation> activations;
 };
 
-partial interface MLGraphBuilder {
-  sequence<MLOperand> lstm(MLOperand input, MLOperand weight, MLOperand recurrentWeight,
-                           unsigned long steps, unsigned long hiddenSize,
-                           optional MLLstmOptions options = {});
-};
-</script>
-<div algorithm=lstm>
+partial interface MLGraphBuilder {
+  sequence<MLOperand> lstm(MLOperand input, MLOperand weight, MLOperand recurrentWeight,
+                           unsigned long steps, unsigned long hiddenSize,
+                           optional MLLstmOptions options = {});
+};
+</script>
+
+{{MLLstmOptions}} has the following members:
+<dl dfn-type=dict-member dfn-for=MLLstmOptions>
+    : <dfn>bias</dfn>
+    ::
+        An {{MLOperand}}. Specifies the 2-D input bias tensor of shape [num_directions, 4 * hidden_size]. The ordering of the bias vectors in the second dimension of the tensor shape is specified according to {{MLLstmOptions/layout}}.
+
+    : <dfn>recurrentBias</dfn>
+    ::
+        An {{MLOperand}}. Specifies the 2-D recurrent bias tensor of shape [num_directions, 4 * hidden_size]. The ordering of the bias vectors in the first dimension of the tensor shape is specified according to {{MLLstmOptions/layout}}.
+
+    : <dfn>peepholeWeight</dfn>
+    ::
+        An {{MLOperand}}. Specifies the 2-D weight tensor for peepholes of shape [num_directions, 4 * hidden_size]. The pack ordering of the weight vectors is for the `input (i)`, `output (o)`, and `forget (f)` gate, respectively.
```

Comment:
> Specifies the 2-D weight tensor for peepholes of shape [num_directions, 4 * hidden_size].

There's an typo error of peepholes shape of the old version Spec, and I've fixed it with PR #385. Please modify the shape to be `[num_directions, 3 * hidden_size]`, thanks.

Reply:
Making a note here, valid to the subsequent comments. 
I think we need to follow a uniform variable naming, and since `hidden_size` is also a method parameter, it should be `hiddenSize`, which pulls all other names to the same convention, when they do exist.